atk/auto/range.rs.html

```diff
@@ -74,7 +74,7 @@
     <span class="doccomment">///</span>
     <span class="doccomment">/// # Implements</span>
     <span class="doccomment">///</span>
-    <span class="doccomment">/// [`RangeExt`](trait.RangeExt.html), [`WidgetExt`](trait.WidgetExt.html), [`glib::object::ObjectExt`](../glib/object/trait.ObjectExt.html), [`BuildableExt`](trait.BuildableExt.html), [`OrientableExt`](trait.OrientableExt.html)</span>
+    <span class="doccomment">/// [`RangeExt`](trait.RangeExt.html), [`WidgetExt`](trait.WidgetExt.html), [`glib::object::ObjectExt`](../glib/object/trait.ObjectExt.html), [`BuildableExt`](trait.BuildableExt.html), [`OrientableExt`](trait.OrientableExt.html), [`WidgetExtManual`](prelude/trait.WidgetExtManual.html), [`BuildableExtManual`](prelude/trait.BuildableExtManual.html)</span>
```

Comment:
This sure  part only for gtkrange.

Reply:
What do you mean?

Reply:
`<record name="Range" c:type="AtkRange" glib:type-name="AtkRange" glib:get-type="atk_range_get_type" c:symbol-prefix="range">`
don't implement any of these interfaces.

Reply:
Then this is an old issue considering that it just added the missing manual ones. Do you have an idea of the origin of this bug? Maybe it's specific to records?

Reply:
Also no this text in atk/vendor.md